docs/devel/automation.md

```diff
@@ -68,92 +70,66 @@ The status of the submit-queue is [online.](http://submit-queue.k8s.io/)
 
 ### Ready to merge status
 
+The submit-queue lists what it believes are required on the [merge requirements tab](http://submit-queue.k8s.io/#/info) of the info page. That may be more up to date.
+
 A PR is considered "ready for merging" if it matches the following:
-   * it has the `lgtm` label, and that `lgtm` is newer than the latest commit
-   * it has passed the cla pre-submit and has the `cla:yes` label
-   * it has passed the travis pre-submit tests
-   * one (or all) of
-      * its author is in kubernetes/contrib/submit-queue/whitelist.txt
-      * its author is in contributors.txt via the github API.
-      * the PR has the `ok-to-merge` label
-   * One (or both of)
-      * it has passed the Jenkins e2e test
-      * it has the `e2e-not-required` label
-
-Note that the combined whitelist/committer list is available at
-[submit-queue.k8s.io](http://submit-queue.k8s.io)
+  * The PR must have the label "cla: yes" or "cla: human-approved"
+  * The PR must be mergeable. aka cannot need a rebase
+  * All of the following github statuses must be green
+     * Jenkins GCE Node e2e
```

Comment:
This only partly a change what's happening today (which is non-sense)

At this very instant:
`e2e-not-required` means that `Jenkins GCE e2e` need never pass.
`Jenkins GCE Node e2e` and `Jenkins unit/integration` still must pass (at least) once.

`e2e-not-required` cause it to not trigger a retest before merge. But if a retest was triggered ALL of e2e, node, and unit/integration had to run (and pass) a second time.

This just makes the policy clear. All must pass once. All or none must pass twice.